src/pages/Browse/index.spec.tsx

```diff
@@ -84,6 +85,57 @@ describe(__filename, () => {
     );
   };
 
+  const setUpVersionFileUpdate = ({
+    extraFileEntries = {},
+    initialPath = 'manifest.json',
+    loadVersionAndFile = true,
+  } = {}) => {
+    const addonId = 9876;
+    const version = {
+      ...fakeVersion,
+      id: 1234,
+      file: {
+        ...fakeVersion.file,
+        entries: {
+          ...fakeVersion.file.entries,
+          ...extraFileEntries,
+        },
+        // eslint-disable-next-line @typescript-eslint/camelcase
+        selected_file: initialPath,
+      },
+    };
+    const store = configureStore();
+
+    if (loadVersionAndFile) {
+      _loadVersionAndFile({ store, version });
+    }
+
+    const fakeThunk = createFakeThunk();
+    const _fetchVersionFile = fakeThunk.createThunk;
+
+    return {
+      _fetchVersionFile,
+      addonId,
+      fakeThunk,
+      store,
+      updateProps: (props = {}) => {
```

Comment:
It would be maybe a bit more explicit to name this function `renderAndUpdateWithProps` or something like that.